frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -146,9 +149,12 @@ export const DetachedTestResultsList: any = ({
               r.patient.middleName,
               r.patient.lastName
             )}
+            <span className="display-block text-gray-30 font-ui-2xs">
```

Comment:
This text color `text-gray-30` is not accessible. Can you change this to `text-base`? It's the lightest accessible gray against a white background. 

My bad for the late review, I didn't know how to view this previously but I see it now.